kable-core/src/appleMain/kotlin/CentralManager.kt

```diff
@@ -45,16 +45,16 @@ public class CentralManager internal constructor(
         }
     }
 
-    private val dispatcher = QueueDispatcher(DISPATCH_QUEUE_LABEL)
+    internal val dispatcher = QueueDispatcher(DISPATCH_QUEUE_LABEL)
     internal val delegate = CentralManagerDelegate()
-    private val cbCentralManager = CBCentralManager(delegate, dispatcher.dispatchQueue, options)
+    private val centralManager = CBCentralManager(delegate, dispatcher.dispatchQueue, options)
```

Comment:
We sure we want this rename? The class here--itself--is named CentralManager. It was actually helpful to see the distinction that `cbCentralManager` was a reference to the Apple API.

Reply:
Fair point. I generally like to be terse in my variable naming, but I see your point: that I've lost useful/valuable information in this name change. I'll revert. 👍